.github/workflows/upload_all.yml

```diff
@@ -81,14 +81,15 @@ jobs:
           release_name="$version_code ($version_name) - $branch@$sha_short"
           echo ::set-output name=release_name::$release_name
       - name: Upload
-        # uses: r0adkll/upload-google-play@9745ef904e395471bca5696056a6ce8a60d18cf8
-        uses: polypoly-eu/upload-google-play@master
+        # uses: r0adkll/upload-google-play@v1.0.16
```

Comment:
Guess you can remove this one now :)

Reply:
Let's leave the comment, so that it's clear which tag we're using, while also locking it to a specific hash.